supported_platforms.md

```diff
@@ -0,0 +1,55 @@
+# Microsoft Azure IoT SDKs for Java
+
+This SDK is tested nightly on Windows 10, Ubuntu 1604, and on an Android emulator. For additional details
+for each tested platform, see the respective sections below. 
+
+While this SDK contains several different packages, each package supports the same platforms and JDKs.
+
+## Supported JDK versions
+
+Currently, this SDK supports JDK versions 8 and 11. This SDK may work with other versions, but there are no guarantees 
```

Comment:
@jamdavi I vaguely remember you validating that our SDK works fine with JDK 11, but can you confirm that for me?